### PR TITLE
build new schema 

### PR DESCRIPTION
**Pull request by ani-sha**
This PR is an enhancement for #12 to print schema. For now, there is no support for a few `GraphQL Types` like `Date`, `Long` etc. 

Sample for running the project :
`jdbc-url : jdbc:teiid:customer@mm://localhost:31000`
`driver name : org.teiid.jdbc.TeiidDriver`

### CODE REVIEW

File: src/main/java/org/graphqlcrudjava/DatabaseConnection.java [hunk 1]
```diff
@@ -3,14 +3,29 @@
...
     //Any Data Source Connection
-        public static Connection connect(String jdbcUrl, String driver, String username, String password) throws SQLException {
+        public static Connection connect() throws SQLException, IOException {
+            System.out.println("Enter JDBC URL : ");
+            String jdbcUrl = bufferedReader.readLine();
+            System.out.println("Enter Suitable Driver : ");
+            String driver = bufferedReader.readLine();
+            System.out.println("Enter your Username : ");
+            String username = bufferedReader.readLine();
+            System.out.println("Enter your Password : ");
```
Comment (wtrocki) [inline src/main/java/org/graphqlcrudjava/DatabaseConnection.java:26]: Let's not print that yet :D

**Comment by wtrocki** [comment]
Amazing change!Really good work! I will see if I can write simple unit tests for it to verify it and add testing to build process tomorrow.

@ani-sha do you think we can architecture this code so we can have this engine independent from database - so we can mock the data layer easily and provide. 

I was checking JDBC mocking libraries as well but wanted to ask you first what would you prefer personally.

**Comment by wtrocki** [comment]
```
Sample for running the project :
jdbc-url : jdbc:teiid:customer@mm://localhost:31000
driver name : org.teiid.jdbc.TeiidDriver
```

Trying to get this running with Postgress but getting strange errors. 
Do you have some docker-compose? 
How to run this? Assume that I'm 5 years old and can execute commands like:

`java -jar target/graphqlcrud-java-1.0.0-SNAPSHOT-runner.jar`

File: src/main/java/org/graphqlcrudjava/DatabaseConnection.java [hunk 2]
```diff
@@ -3,14 +3,29 @@
...
 public class DatabaseConnection {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseConnection.class);
 
+    private static  final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
     //Any Data Source Connection
-        public static Connection connect(String jdbcUrl, String driver, String username, String password) throws SQLException {
+        public static Connection connect() throws SQLException, IOException {
+            System.out.println("Enter JDBC URL : ");
```
Comment (wtrocki) [inline src/main/java/org/graphqlcrudjava/DatabaseConnection.java:20]: I would preffer if you will use environment variables here and document them as part of the PR.

**Comment by rareddy** [comment]
> Do you think we can architecture this code so we can have this engine independent from database

This is great point, I mentioned this before to @ani-sha probably lost in translation. Maybe I can help with arranging the code a little bit for that to the separation of concerns. Can I merge and start on that if everyone is Ok?

**Comment by machi1990** [comment]
> > Do you think we can architecture this code so we can have this engine independent from database
> 
> This is great point, I mentioned this before to @ani-sha probably lost in translation. Maybe I can help with arranging the code a little bit for that to the separation of concerns. Can I merge and start on that if everyone is Ok?

Okay with me.

**Comment by wtrocki** [comment]
Same for me. Thanks for merging.

**Pull request merged**